listenbrainz/mbid_mapping/mapping/year_mapping.py

```diff
@@ -0,0 +1,229 @@
+import psycopg2
```

Comment:
Is this mostly the same as mbid_mapping.py, except with a slightly different initial query sort order? Can all of the common code be shared?

Reply:
There are more subtle far-reaching changes than just that, but there is some code overlap that can be consolidated in a future PR.